### PR TITLE
[doc] Noted blocklisted files are quarantined

### PR DESCRIPTION
**Pull request by rseldner**
Explicitly noted block listed files are quarantined

Previews:
- ESS: [Configure an integration policy yada yada](https://security-docs_bk_5918.docs-preview.app.elstc.co/guide/en/security/master/configure-endpoint-integration-policy.html#manage-quarantined-files)
- Serverless: Navigate to the same page in [serverless docs](https://github.com/elastic/security-docs/pull/5918#issuecomment-2417004586)

### CODE REVIEW

**Review by gabriellandau — DISMISSED** [review]

File: docs/getting-started/configure-integration-policy.asciidoc [hunk 1]
```diff
@@ -84,7 +84,7 @@ image::images/install-endpoint/malware-protection.png[Detail of malware protecti
 [[manage-quarantined-files]]
 === Manage quarantined files
 
-When *Prevent* is enabled for malware protection, {elastic-defend} will quarantine any malicious file it finds. Specifically {elastic-defend} will remove the file from its current location, encrypt it with the encryption key `ELASTIC`, move it to a different folder, and rename it as a GUID string, such as `318e70c2-af9b-4c3a-939d-11410b9a112c`.
+When *Prevent* is enabled for malware protection, {elastic-defend} will quarantine any malicious file it finds (this includes files defined the <<blocklist>>). Specifically {elastic-defend} will remove the file from its current location, encrypt it with the encryption key `ELASTIC`, move it to a different folder, and rename it as a GUID string, such as `318e70c2-af9b-4c3a-939d-11410b9a112c`.
```
Comment (joepeeples) [inline docs/getting-started/configure-integration-policy.asciidoc:87]:
```suggestion
When *Prevent* is enabled for malware protection, {elastic-defend} will quarantine any malicious file it finds (this includes files defined in the <<blocklist>>). Specifically {elastic-defend} will remove the file from its current location, encrypt it with the encryption key `ELASTIC`, move it to a different folder, and rename it as a GUID string, such as `318e70c2-af9b-4c3a-939d-11410b9a112c`.
```

**Comment by joepeeples** [comment]
Thanks for this update, @rseldner! I added a commit to update the same content in our serverless docs, and also added some labels to trigger backports to several recent versions. I'll handle merging and everything from here on out.

**Review by joepeeples — APPROVED** [review]

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-kqr17wklr-elastic-dev.vercel.app

* Built with commit edd443ff92f0b1131f1cf5e21e91b540967d679f

> Issues? Visit #next-docs in Slack

**Review by benironside — APPROVED** [review]
LGTM, super trivial thing is that the link "blocklist" is in italics on the serverless version

**Pull request merged**